rosdep/python.yaml

```
@@ -6315,6 +6315,13 @@ python3-fpdf2-pip:
   ubuntu:
     pip:
       packages: [fpdf2]
+python3-freezegun:
+  debian: [python3-freezegun]
+  fedora: [python3-freezegun]
+  gentoo: [dev-python/freezegun]
+  opensuse: [python3-freezegun]
+  rhel: [python3-freezegun]
```

It is only available on RHEL 9? https://pkgs.org/search/?q=python3-freezegun

If so, since we also currently support RHEL 8, this should be:

```suggestion
  rhel:
    '*': [python3-freezegun]
    '8': null
```

---

My bad, the search only shows RHEL 9 results by default :facepalm: